config/locales/en/manage_competencies.yml

```diff
@@ -10,11 +10,21 @@ en:
         heading: Manage competencies
       table_cells:
         no_competencies: No competencies
+        history: History
+    history:
+      show:
+        page_title: Competencies history
```

Comment:
This might be clearer as User's Competency History as thats what we are referring to in the heading.